lib/utils.js

```diff
@@ -91,31 +134,31 @@ function Utils(config) {
             throw new Error('Unlocated features can not be stored.');
         }
 
-        var info = Metadata(config.dyno, dataset).getFeatureInfo(f);
-        var buf = geobuf.featureToGeobuf(f).toBuffer();
-        var tile = tilebelt.bboxToTile([info.west, info.south, info.east, info.north]);
-        var cell = tilebelt.tileToQuadkey(tile);
+        var bounds = extent(f);
+        var info = {
+            bounds: bounds,
+            west: bounds[0],
+            south: bounds[1],
+            east: bounds[2],
+            north: bounds[3]
+        };
+        var buf = utils.encodeFeature(f);
         var useS3 = buf.length >= config.MAX_GEOMETRY_SIZE;
         var s3Key = [config.prefix, dataset, primary, +new Date()].join('/');
         var s3Params = { Bucket: config.bucket, Key: s3Key, Body: buf };
 
-        var item = {
-            dataset: dataset,
-            id: 'id!' + primary,
-            cell: 'cell!' + cell,
-            size: info.size,
-            west: truncateNum(info.west),
-            south: truncateNum(info.south),
-            east: truncateNum(info.east),
-            north: truncateNum(info.north)
-        };
-
-        if (useS3) {
-            item.s3url = ['s3:/', config.bucket, s3Key].join('/');
-            return [item, s3Params];
-        } else {
-            item.val = buf;
-            return [item];
+        var databaseFeature = utils.createFeatureKey(dataset, primary);
+        databaseFeature.val = useS3 ? null : buf;
+        databaseFeature.size = buf.length;
+        databaseFeature.west = truncateNum(info.west);
```

Comment:
Since we don't have a BBOX query here anymore (moving to another module / index in ultimately), is there any reason for our main table features to contain any more information than id and geobuf?